src/reactnative/react-native-init.ts

```diff
@@ -127,19 +127,6 @@ export const reactNativeInit = {
       })
       .then(navigation => {
         if (navigation.navigation === "none") return detect.packageName();
-        settings.components.navigation = {
```

Comment:
'k versta nie direct waarom dit weg mag?

Reply:
navigation is geen folder waar we een "normaal" component in kunnen toevoegen, enkel navigations

De mogelijkheid om navigations met Planter aan te maken moet nog ontwikkeld worden.